Documentation/API Backend/openrpc/methods/tags/motion/jogJoint.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "jogJoint",
+    "description": "Jog an joint with a specified velocity",
+    "params": [
+        {
+            "name": "jointIndex",
+            "description": "The index of the joint to jog",
+            "schema": {
+                "type": "integer"
+            }
+        },
+        {
+            "name": "direction",
+            "description": "Direction to jog the joint.",
```

Comment:
yes. That makes sense

Reply:
You have to be careful with the velocity field. There are other factors that contribute to the final velocity of the jog. Most of these factors are determined in the safety circuit. For example if a specific safety switch is open (like a person gate) than the max jog speed of the robot is radically lowered and would be outside of whatever value you sent to the API. 

In that respect. I believe the velocity value should instead be a percentage.

Reply:
I guess that would depend on what you expected to happen if something changed the maximum permissible speed while the robot was moving, for example if the control setpoint is 20% and robot is moving at 20mm/s then the safety system changes the limit from 100mm/s to 50mm/s, would you expect the robot to drop back to 10mm/s or remain constant at 20mm/s.

We'd probably also want a separate absolute interface for use with external control systems (ROS etc) - perhaps that would be enabled in addition to the jog interface if the client has an extra permission enabled, since it shouldn't be accessible from the UI as it'd be operational in run mode instead of teach/manual mode.